carbon/carbon.py

```diff
@@ -38,47 +34,21 @@ def _create_carbon_url(code, **carbon_options: str) -> str:
 
 def create_code_image(code: str, **kwargs: str) -> None:
     """Generate a beautiful Carbon code image"""
-    # options = Options()
-    # if not bool(kwargs.get("interactive", False)):
-    #     options.add_argument("--headless")
-
-    # service = (
-    #     Service(executable_path=kwargs["driver_path"])
-    #     if kwargs["driver_path"]
-    #     else Service()
-    # )
-
     destination = kwargs.get("destination", os.getcwd())
-    prefs = {"download.default_directory": destination}
-    # options.add_experimental_option("prefs", prefs)
+    headless = kwargs.get("headless", True)
     
-
-    if kwargs.get("disable-dev-shm", False):
-        options.add_argument("disable-dev-shm-usage")
-
     with sync_playwright() as p:
-        browser = p.chromium.launch(headless=False)
-        
-        context = browser.new_context()
-        
-        page = context.new_page()
-        url = _create_carbon_url(code, **kwargs)
-        page.goto(url)
-
-        page.click("#export-menu")
-        page.click("#export-png")
-
-        download = page.wait_for_event("download")
-        download_path = os.path.join(destination, "carbon_image.png")
-        download.save_as(download_path)
-        
-        sleep(SECONDS_SLEEP_BEFORE_DOWNLOAD)
-
-        browser.close()
-
-    # with webdriver.Chrome(service=service, options=options) as driver:
-        # driver.get(url)
-        # driver.find_element(By.ID, "export-menu").click()
-        # driver.find_element(By.ID, "export-png").click()
-        # make sure it has time to download the image
-        
+        with p.chromium.launch(headless=headless) as browser:
+            context = browser.new_context()
+            page = context.new_page()
+            url = _create_carbon_url(code, **kwargs)
+            page.goto(url)
+
+            page.click("#export-menu")
+            page.click("#export-png")
+
+            download = page.wait_for_event("download")
+            download_path = os.path.join(destination, "carbon_image.png")
+            download.save_as(download_path)
+            
+            sleep(SECONDS_SLEEP_BEFORE_DOWNLOAD)
```

Comment:
do we still need this to be configurable? I think a hardcoded value would be fine for now?